examples/webgl-vector-layer.js

```diff
@@ -0,0 +1,65 @@
+import GeoJSON from '../src/ol/format/GeoJSON.js';
+import Layer from '../src/ol/layer/Layer.js';
+import Map from '../src/ol/Map.js';
+import OSM from '../src/ol/source/OSM.js';
+import TileLayer from '../src/ol/layer/WebGLTile.js';
+import VectorSource from '../src/ol/source/Vector.js';
+import View from '../src/ol/View.js';
+import WebGLVectorLayerRenderer from '../src/ol/renderer/webgl/VectorLayer.js';
+import {asArray} from '../src/ol/color.js';
+import {packColor} from '../src/ol/renderer/webgl/shaders.js';
+
+class WebGLLayer extends Layer {
+  createRenderer() {
+    return new WebGLVectorLayerRenderer(this, {
+      className: this.getClassName(),
```

Comment:
The `className` option is unused, so I pushed a commit that removes it from the example.